app/graphql/resolvers/create_article.rb

```diff
@@ -31,6 +32,7 @@ def call(_obj, args, ctx)
         issue: args["issue"],
         summary: args["summary"],
         created_at: args["created_at"],
+        is_published: args["is_published"] || true
```

Comment:
Do you really want to default to true?

Reply:
Eventually I'll remove or change the default, but for now I want code currently in use to work without modifications.